ivy/functional/ivy/elementwise.py

```diff
@@ -2418,7 +2418,7 @@ def less_equal(
     `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.tan.html>`_ # noqa
     in the standard.
 
-     Functional Examples
+     Examples
     -------------------
```

Comment:
The number of `-` should be equal to the number of characters in the header above it.